source/includes/language-compatibility-table-csharp.rst

```diff
@@ -205,6 +220,10 @@
 Versions 1.10 and 1.11 of the driver are compatible with .NET 3.5
 through 4.8 only.
 
-.. [#atlas-connection] When using .NET 5, you can't connect to Atlas clusters running MongoDB 4.0 due to a certificate issue.
+.. [#atlas-connection] When using .NET 5/6, you can't connect to Atlas clusters running MongoDB 4.0 due to a certificate issue. This does not impact clusters running MongoDB 4.2+.
 
 .. [#4.5.2] .NET/C# driver versions 2.8 to 2.13 requires .NET Framework 4.5.2 or higher.
+
+.. [#2.14-note] Version 2.14 is compatible with .NET 4.7.2 and .NET 4.8 and .NET Core 2.0
```

Comment:
>.NET/C# driver versions 2.14 requires .NET Framework 4.7.2 or higher.

Also move this above [#4.5.2] so the numbering is sequential on the table.

Reply:
Change:

>Version 2.14 is compatible with .NET 4.7.2 and .NET 4.8 and .NET Core 2.0

to:

>.NET/C# driver versions 2.14 requires .NET Framework 4.7.2 or higher.

so that it matches the [#4.5.2] note.